include/vigra/blockwise_labeling.hxx

```diff
@@ -137,7 +137,7 @@ template <class DataBlocksIterator, class LabelBlocksIterator,
           class Equal, class Mapping>
 typename BlockwiseLabelingResult<LabelBlocksIterator>::type
 blockwiseLabeling(DataBlocksIterator data_blocks_begin, DataBlocksIterator data_blocks_end,
-                  LabelBlocksIterator label_blocks_begin, LabelBlocksIterator label_blocks_end,
+                  LabelBlocksIterator label_blocks_begin, LabelBlocksIterator,
```

Comment:
It the two ranges must have the same length, don't ignore the end iterator but add a precondition instead.